perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -281,7 +296,11 @@ sub dashboard_export_problems_add_columns {
             container_req_type => $report->get_extra_field_value('Container_Request_Container_Type') || '',
             container_req_reason => $report->get_extra_field_value('Container_Request_Reason') || '',
             missed_collection_id => $report->get_extra_field_value('service_id') || '',
-        }
+        };
+
+        %$data = (%$data, %bulky_items);
```

Comment:
If you moved the bulky_items code down here, you could add it to $data without needing the intermediary, perhaps?